mappings/net/minecraft/client/render/model/ModelLoader.mapping

```diff
@@ -10,11 +10,15 @@ CLASS net/minecraft/class_1088 net/minecraft/client/render/model/ModelLoader
 	FIELD field_21558 SHIELD_BASE_NO_PATTERN Lnet/minecraft/class_4730;
 	FIELD field_21772 BLOCK_DESTRUCTION_RENDER_LAYERS Ljava/util/List;
 	FIELD field_21774 spriteAtlasManager Lnet/minecraft/class_4724;
+	FIELD field_32986 BUILTIN Ljava/lang/String;
+	FIELD field_32987 BUILTIN_GENERATED Ljava/lang/String;
+	FIELD field_32988 BUILTIN_ENTITY Ljava/lang/String;
+	FIELD field_32989 MISSING Ljava/lang/String;
```

Comment:
Should these be `XXX_NAME` or `XXX_PATH` as they are paths of a few builtin identifiers?